packages/features/ee/managed-event-types/hooks/useLockedFieldsManager.tsx

```diff
@@ -1,27 +1,102 @@
 // eslint-disable-next-line no-restricted-imports
```

Comment:
Just feedback on the naming, as this file doesn't only include the hook file this may be confusing.

Reply:
Oops, a remnant from earlier changes 🤦 
Nice catch